docker/local.cfg

```diff
@@ -1,3 +1,4 @@
 # This is a sample config file for backend.
 # Replace with production local.cfg.
 # It will be mounted to docker container.
+rest.cors.allowed-origins = ${dspace.ui.url}, http://localhost:3000/, http://127.0.0.1:3000, http://127.0.0.1:4000
```

Comment:
I guess this is good for development, but we should have it as the default

Reply:
agreed